### DISCUSSION TITLE
ROSES Open-Source Tools, Frameworks, and Libraries Selection

### DISCUSSION THREAD

**Discussion by LeStarch** [nasa/fprime#3041] [Announcements]
Exciting news!

F Prime was recently selected to receive funding for a proposal called “How To F Prime: Empowering Science Missions Through Documentation and Examples.”  

https://www.nasa.gov/news-release/nasa-funds-open-source-software-underpinning-scientific-innovation/

As part of this work, we need to solicit opinions from the larger community. Please let us know what patterns you'd like to see documented!

To submit an pattern idea please [create a new pattern request](https://github.com/nasa/fprime/discussions/new?category=pattern-request).

To vote on patterns you wish to see, please "upvote" [pattern requests that have been submitted](https://github.com/nasa/fprime/discussions/categories/pattern-request) that you and your team would find helpful.

**Comment by SiboVG** [comment]
Great, congrats! Is there a timeline for the proposal?

**Reply by LeStarch** [reply]
The work funded by this proposal is intended to kick-off in January 2025.  Our proposal is a 3-year award.  This is all the information we have at this time.
